src/components/elements/heading/heading.types.ts

```diff
@@ -1,7 +1,7 @@
 import type { FontWeigth, TextColor } from '@/styles';
 
 type headingSize = 'xs' | 's' | 'l' | 'm' | 'xl';
-type headingLevel = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+type headingLevel = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'p';
```

Comment:
Can I ask why this addition? I don't think we should support the p tag here, the component is only intended to be used as a Heading with H* tags. Is for that we have the Text component.

Reply:
But it is also for styling, I don't think these cards had H* tags, but we can as Dima if this cards are intented to be an H4 for example

Reply:
Sure, let's ask Dima about the heading tag, but we should still not add the p tag here and instead use the Text element. You can extend the Text styled in the card module if you need it. Take a look here: https://styled-components.com/docs/basics#extending-styles

Reply:
@nephlin7 Dime confirms me that we should use an H3